uni/lib/remorse.icn

```diff
@@ -0,0 +1,157 @@
+############################################################################
+#
+#	File:	  remorse.icn
+#
+#	Subject:  Procedures to convert a string to and from Morse code
+#
+#	Author:	  Don Ward
+#
+#	Date:	  October 2022
+#
+############################################################################
+#
+# This is a re-implementation of morse.icn which, compared to its predecessor:
```

Comment:
should we add the full path to morse.icn (or say IPL version)  to let the reader know which file you are referring to